src/app/components/annotations/pages/verification/verification.component.ts

```diff
@@ -246,18 +234,107 @@ class VerificationComponent
     this.hasUnsavedChanges = false;
   }
 
+  protected handleDecision(decisionEvent: Event): void {
+    if (!this.isDecisionEvent(decisionEvent)) {
+      console.error("Received invalid decision event", decisionEvent);
+      return;
+    }
+
+    this.hasUnsavedChanges = true;
```

Comment:
i don't know where this is unset, but the service requests are fire and forget... so?

Reply:
The page doesn't currently alert if no decisions have been made. If any decisions have been made, the page will always alert.

This is a bit of a remanence from before users could resume verifications, so I don't think it's as needed these days.